scripts/core_uploader.sh

```diff
@@ -0,0 +1,97 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# 	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+# This script uploads the following to the provided S3 bucket under S3_KEY_PREFIX
+#   <s3_key>_<date>_HOSTNAME_RANDOM.core.zip
+#   <s3_key>_<date>_HOSTNAME_RANDOM.executable
+#   <s3_key>_<date>_HOSTNAME_RANDOM.stacktrace
+#   <s3_key>_<date>_HOSTNAME_RANDOM.all.zip
+
+
+export S3_BUCKET=$1
+export S3_KEY_PREFIX=$2
+export CRASH_FOLDER="/cores-out"
+
+# Folder Structure
+# $CRASH_FOLDER: All the crash symbol information is combined and added to this folder.
+#    It is uploaded to s3 and added to the /cores folder
+# /cores: Files are sent to the cores folder
```

Comment:
but its now /cores-out ?

Reply:
Right... So if you mount /cores to some EFS Volume, the problem is that there may be other cores already there, so you can't expect that folder to be empty

Reply:
Sorry I don't understand what you are saying. Does this change the EFS core tutorial I wrote or not?

Reply:
No, https://github.com/aws/aws-for-fluent-bit/pull/548/files#diff-ac4a590fd0373a499e5f24246e9c6b554b57cf458124af8e48d471ed9ba48128R85. We do all the processing in the /cores-out folder, but in the end we copy all the files to the /cores folder which is what you might mount an EFS volume to...


The goal is to make sure that this folder is empty when you start. If you mount an efs volume, it might not be empty.

Reply:
Ah... makes sense